### ISSUE TITLE
Projects App - misc ui/ux issues 

### ISSUE THREAD

**Issue by ottodevs** [AutarkLabs/planning-suite#476]
RELATED TICKET (review this first) : https://github.com/AutarkLabs/planning-suite/issues/480

YALDA UPDATE:
I am checking off the items where the tickets were created...

> * [x]  Issues view shows "Allocated" after assignment is requested instead of after assignment
> * [x]  Should say "Unassigned/Assigned" instead of "Unallocated/Allocated"
> * [x]  Only show the quantity available if the qty is greater than 1
> * [x]  Submitting work causes the issues to show as "unallocated"
> * [x]  The note about Github comments is still showing in the submitWork panel
> * [x]  The Review Work context menu option shows before any work is submitted (and leads to a js stack trace if clicked before work is submitted)
> * [x]  pictures and links from github issue descriptions don't render (likely requires a new issue)

**Comment by stellarmagnet** [comment]
Created a bunch of child issues, so closed this one.
